uvloop/loop.pyx

```diff
@@ -645,25 +645,17 @@ cdef class Loop:
 
         cdef:
             UVStream stream
-            int queued_len
 
-        if UVLOOP_DEBUG:
-            queued_len = len(self._queued_streams)
+        streams = list(self._queued_streams)
```

Comment:
Can we have 2 python `list` objects that are always allocated and switch them here?

```
streams = self._queued_streams
self._queued_streams = self._queued_streams2
try:
   ...
finally:
   streams.clear()
```

Reply:
Sure, only that it'll be 2 `set`s.